output/elastic/outputelastic.go

```diff
@@ -105,8 +105,15 @@ func InitHandler(ctx context.Context, raw *config.ConfigRaw) (config.TypeOutputC
 	// map Printf to error level
 	logger := &errorLogger{logger: goglog.Logger}
 
+	// replace env var names with values on URL config
+	var resolvedURLs []string
+	for _, url := range conf.URL {
+		newURL := logevent.FormatWithEnv(url)
+		resolvedURLs = append(resolvedURLs, newURL)
```

Comment:
In order to check resolvedURLs in unit test, you can save them in OutputConfig struct as a new field.